.gitignore

```diff
@@ -23,4 +23,8 @@ out/
 .idea_modules/
 *.iml
 *.ipr
-*.iws
\ No newline at end of file
+*.iws
+/src/odyssey/java/assignments/arrays/w3School/W3array_33.java
+/src/odyssey/java/assignments/arrays/w3School/W3array_35.java
+/src/odyssey/java/assignments/arrays/w3School/W3array_40.java
+/src/odyssey/java/assignments/arrays/w3School/W3array_41.java
```

Comment:
Why did you add this to gitignore file

Reply:
i need to work on this little more. I will update ASAP i finish this three